<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>232</y>
        <width>93</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Pepsi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>171</y>
        <width>130</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1.3 DT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>110</y>
        <width>130</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1.3 DT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>18</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>STORE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>232</y>
        <width>130</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1.7 DT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>178</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>238</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ViewCart">
      <property name="geometry">
       <rect>
        <x>278</x>
        <y>378</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>293</y>
        <width>101</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Sprite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>298</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>171</y>
        <width>105</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Fanta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>458</x>
        <y>118</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>293</y>
        <width>130</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1.5 DT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>82</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Coke</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>label_7</zorder>
     <zorder>label_6</zorder>
     <zorder>label</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_3</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>label_9</zorder>
     <zorder>label_2</zorder>
     <zorder>ViewCart</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>26</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>24</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Payment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Confirm">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="PricePay">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>121</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>121</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>121</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>151</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Card N°</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditName">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditConfirm">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>131</y>
        <width>317</width>
        <height>83</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>48</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Thank You</string>
      </property>
     </widget>
     <widget class="QPushButton" name="New">
      <property name="geometry">
       <rect>
        <x>201</x>
        <y>219</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="toolTipDuration">
      <number>-4</number>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>24</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Items in Cart</string>
      </property>
     </widget>
     <widget class="QListWidget" name="CART">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>121</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="TotalPrice">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>121</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>28</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Pay">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>410</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackToSelection">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
